src/main/java/org/datadog/jenkins/plugins/datadog/model/BuildPipeline.java

```diff
@@ -144,6 +144,8 @@ private void completeInformation(final List<BuildPipelineNode> nodes, final Buil
             // Propagate error to all parent stages
             if(node.isError() && !parent.isError()) {
                 propagateResultToAllParents(node, "error");
+            } else if(node.isUnstable() && !parent.isUnstable()) {
+                propagateResultToAllParents(node, "unstable");
```

Comment:
Should we make this a constant, perhaps? There're 12 places where this literal is used